src/lib/holocene/navigation/nav-row.svelte

```diff
@@ -4,28 +4,34 @@
   export let noFilter = false;
   export let wrap = false;
   export let externalLink = false;
+  export let handleClick: () => void = null;
   let classes = `nav-row ${$$props.class} ${isCloud ? 'cloud' : 'local'}`;
 </script>
 
-<li
-  class={link ? '' : classes}
-  class:noFilter
-  class:wrap
-  data-cy={$$props['data-cy']}
-  on:click
->
-  {#if link && externalLink}
-    <a href={link} target="_blank" class={classes}>
-      <slot />
-    </a>
-  {:else if link}
-    <a href={link} class={classes}>
-      <slot />
-    </a>
-  {:else}
+{#if handleClick !== null}
+  <button on:click={handleClick} class="{classes} text-left">
     <slot />
-  {/if}
-</li>
+  </button>
+{:else}
+  <li
```

Comment:
What do you think about moving the `button` inside the `li` element here?
```suggestion
  <li
  class={link ? '' : classes}
  class:noFilter
  class:wrap
  data-cy={$$props['data-cy']}
>
  {#if handleClick !== null}
    <button on:click={handleClick} class="{classes} text-left">
      <slot />
    </button>
  {:else if link && externalLink}
    <a href={link} rel="noreferrer" target="_blank" class={classes}>
      <slot />
    </a>
  {:else if link}
    <a href={link} class={classes}>
      <slot />
    </a>
  {:else}
    <slot />
  {/if}
</li>
```

Reply:
Similar to using NavTooltip in data-encoder-status I think this caused some unintended behavior with the nav animation. I'll double check though, there might be an easy fix 👍

Reply:
This works great, I'll make this change 🎉